polyply/src/linalg_functions.py

```diff
@@ -235,3 +235,38 @@ def _rotate_xyz(object_xyz, theta_x, theta_y, theta_z):
     return rotated_object
 
 rotate_xyz = jit(_rotate_xyz)
+
+def not_exceeds_max_dimensions(point, maxdim):
+    """
+    Check if point is within the compontents of
+    the maxdim vector. Note that all coodinates in
+    polyply are definiet positive.
```

Comment:
typo
Also semi-positive definite, since 0 is also fine.

Reply:
good catch didn't update the doc-string